internal/services/oracle/oracle_autonomous_database_cross_region_disaster_recovery_data_source.go

```diff
@@ -0,0 +1,543 @@
+	PeerDbIds                                     []string `tfschema:"peer_database_ids"`
+	Preview                                       bool     `tfschema:"preview"`
+	PreviewVersionWithServiceTermsAccepted        bool     `tfschema:"preview_version_with_service_terms_accepted"`
+	PrivateEndpoint                               string   `tfschema:"private_endpoint"`
+	PrivateEndpointIP                             string   `tfschema:"private_endpoint_ip"`
+	PrivateEndpointLabel                          string   `tfschema:"private_endpoint_label"`
+	ProvisionableCPUs                             []int64  `tfschema:"provisionable_cpus"`
+	RemoteDataGuardEnabled                        bool     `tfschema:"remote_data_guard_enabled"`
+	ServiceConsoleUrl                             string   `tfschema:"service_console_url"`
+	SqlWebDeveloperUrl                            string   `tfschema:"sql_web_developer_url"`
+	SubnetId                                      string   `tfschema:"subnet_id"`
+	TimeCreatedInUtc                              string   `tfschema:"time_created_in_utc"`
+	TimeDataGuardRoleChangedInUtc                 string   `tfschema:"time_data_guard_role_changed_in_utc"`
+	TimeDeletionOfFreeAutonomousDatabaseInUtc     string   `tfschema:"time_deletion_of_free_autonomous_database_in_utc"`
+	TimeLocalDataGuardEnabledInUtc                string   `tfschema:"time_local_data_guard_enabled_in_utc"`
+	TimeMaintenanceBeginInUtc                     string   `tfschema:"time_maintenance_begin_in_utc"`
+	TimeMaintenanceEndInUtc                       string   `tfschema:"time_maintenance_end_in_utc"`
+	TimeOfLastFailoverInUtc                       string   `tfschema:"time_of_last_failover_in_utc"`
+	TimeOfLastRefreshInUtc                        string   `tfschema:"time_of_last_refresh_in_utc"`
+	TimeOfLastRefreshPointInUtc                   string   `tfschema:"time_of_last_refresh_point_in_utc"`
+	TimeOfLastSwitchoverInUtc                     string   `tfschema:"time_of_last_switchover_in_utc"`
+	TimeReclamationOfFreeAutonomousDatabaseInUtc  string   `tfschema:"time_reclamation_of_free_autonomous_database_in_utc"`
+
+	UsedDataStorageSizeInGb int64    `tfschema:"used_data_storage_size_in_gb"`
+	UsedDataStorageSizeInTb int64    `tfschema:"used_data_storage_size_in_tb"`
+	VnetId                  string   `tfschema:"virtual_network_id"`
+	AllowedIpAddresses      []string `tfschema:"allowed_ip_addresses"`
+}
+
+func (d AutonomousDatabaseCrossRegionDisasterRecoveryDataSource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"resource_group_name": commonschema.ResourceGroupNameForDataSource(),
+
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: validate.AutonomousDatabaseName,
+		},
+	}
+}
+
+func (d AutonomousDatabaseCrossRegionDisasterRecoveryDataSource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
```

Comment:
please reorder the fields with the [guidance](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-new-data-source.md#step-3-scaffold-an-emptynew-data-source)

<img width="767" height="214" alt="Image" src="https://github.com/user-attachments/assets/aff17d07-1f19-43f7-a51b-005daf5c77f6" />